ios/FluentUI/Tab Bar/TabBarView.swift

```diff
@@ -40,10 +40,15 @@ open class TabBarView: UIView {
                 preconditionFailure("tab bar items can't be more than \(Constants.maxTabCount)")
             }
 
-            for item in items {
+            for (index, item) in items.enumerated() {
                 let tabBarItemView = TabBarItemView(item: item, showsTitle: showsItemTitles)
                 let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleTabBarItemTapped(_:)))
                 tabBarItemView.addGestureRecognizer(tapGesture)
+
+                // seems like iOS 14 `.tabBar` accessibilityTrait doesn't seem to read out the index automatically
+                if #available(iOS 14.0, *) {
+                    tabBarItemView.accessibilityHint = String(format: "Accessibility.TabBarItemView.Hint".localized, index + 1, numberOfItems)
```

Comment:
I think it won't have a practical difference in the outcome here since the index is an integer, but we're using [localizedStringWithFormat:](https://developer.apple.com/documentation/foundation/nsstring/1497301-localizedstringwithformat) as a standard for these calls.

For example: https://github.com/microsoft/fluentui-apple/blob/4bd410eef400cb1340bec4b18122cf89b47dfef6/ios/FluentUI/SegmentedControl/SegmentedControl.swift#L624

Reply:
Same in line 283.